lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -155,7 +155,11 @@ def is_initialized(self) -> bool:
 
     def is_sealed(self) -> bool:
         """Return whether Vault is sealed."""
-        return self._client.sys.is_sealed()
+        try:
```

Comment:
Why does the change here not come with a version bump?

Reply:
This change is actually [done in the k8s charm](https://github.com/canonical/vault-k8s-operator/pull/403), I need to pull it over now that it is actually merged.

Reply:
https://github.com/canonical/vault-operator/pull/215/commits/ce42ebf9224ef6bb0b1b030a298739ab5ba65c80

Reply:
https://github.com/canonical/vault-operator/pull/215/commits/ba626cb4d15b9028c65e799894a5d5fa4a5ea05c